templates/definition/meter/solax-inverter-cloud.yaml

```diff
@@ -2,18 +2,18 @@ template: solax-inverter-cloud
 products:
   - brand: Solax
     description:
-      de: Wechselrichter
-      en: Inverter
+      de: PV-Wechselrichter (Cloud)
```

Comment:
Dito. Sonst: "Wechselrichter".

Reply:
Doch, es gibt z. B. auch reine Batteriewechselrichter (z. B. Sunny Island, Sunny Boy Storage, …)
Ich will das vor allem wegen `configure` und der Doku eindeutiger machen da dort in der Auswahl der Hintergrund fehlt.
Die anderen Templates muss ich dann auch noch anpassen wo nötig.